src/main/java/org/venice/piazza/servicecontroller/Application.java

```diff
@@ -113,6 +118,16 @@ public static void inspectSprintEnv(ApplicationContext ctx) {
 		}
 	}
 
+	@Bean(name = "UpdateJobsQueue")
```

Comment:
Why is the name field necessary here? It is not used in other modules. Would recommend trying to stay consistent.